app/containers/NewRequestPage.js

```diff
@@ -13,7 +13,12 @@ class NewRequestPage extends Component {
   }
 
   componentDidMount() {
-    this.setState({request: new Request({method: 'GET'})});
+    const newRequest = new Request({
+      method: 'GET',
+      projectId: this.props.params.projectId
```

Comment:
This seems to introduce a slight regression where a delete button is shown on the 'new request' screen.

(Why) do we need this?

Also this really hints to the fact that determining when to display a delete button is not very robust currently.